src/aics-image-viewer/components/Toolbar/styles.css

```diff
@@ -67,77 +67,56 @@
 .viewer-toolbar {
   padding: 12px;
   text-align: center;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
 
-  .viewer-toolbar-left,
-  .viewer-toolbar-right {
+  & > div {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    vertical-align: middle;
+  }
+
+  & .viewer-toolbar-left,
+  & .viewer-toolbar-right {
     position: absolute;
     &:not(:empty) {
       padding: 0 12px;
     }
   }
-  .viewer-toolbar-left {
+  & .viewer-toolbar-left {
     left: 0;
   }
-  .viewer-toolbar-right {
+  & .viewer-toolbar-right {
     right: 0;
   }
 
-  .viewer-toolbar-group:not(:last-child) {
+  & .viewer-toolbar-group {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    vertical-align: middle;
+  }
+
+  & .viewer-toolbar-group:not(:last-child) {
     margin-right: 14px;
   }
-  .viewer-toolbar-group > :not(:last-child) {
+  & .viewer-toolbar-group > :not(:last-child) {
     margin-right: 6px;
   }
 
   /* Icon-only buttons */
 
-  .ant-btn-icon-only {
-    padding-bottom: 1px;
+  & .ant-btn-icon-only {
+    padding-bottom: 4px;
+    padding-left: 1px;
   }
 
-  .btn-borderless {
-    background-color: transparent;
+  & .btn-borderless {
+    background-color: var(--color-toolbar-button-bg);
     border-color: transparent;
-    color: #bfbfbf;
-
-    > .ant-btn[disabled] {
-      background-color: transparent;
-      border-color: transparent;
-      color: #bfbfbf;
-      padding: 0;
-    }
-
-    &.btn-active,
-    &:hover {
-      outline: none;
-      box-shadow: none;
-      color: white;
-      border-color: $light-gray;
-    }
-
-    &.btn-active {
-      background-color: #4b4b4b;
-    }
-
-    &:hover {
-      background-color: $highlight-purple;
-    }
-
-    &:focus-within:not(.btn-active):not(:hover) {
-      background-color: transparent;
-      border-color: transparent;
-      color: #bfbfbf;
-    }
-
-    &[ant-click-animating-without-extra-node="true"]::after {
-      box-shadow: none !important;
-      background-color: #222222;
-    }
-  }
-
-  span.btn-borderless {
-    background-color: transparent !important;
-    border-color: transparent !important;
```

Comment:
Styling now mostly managed by `StyleProvider`. Icon buttons behave the same as other buttons when hovered/clicked.